include/global.fieldmap.h

```diff
@@ -293,6 +293,15 @@ enum {
 #define PLAYER_AVATAR_FLAG_FISHING    (1 << PLAYER_AVATAR_STATE_FISHING)
 #define PLAYER_AVATAR_FLAG_DASH       (1 << PLAYER_AVATAR_STATE_WATERING)
 
+enum {
+    PLAYER_AVATAR_GFX_NORMAL,
+    PLAYER_AVATAR_GFX_BIKE,
+    PLAYER_AVATAR_GFX_SURF,
+    PLAYER_AVATAR_GFX_FIELD_MOVE,
+    PLAYER_AVATAR_GFX_FISH,
+    PLAYER_AVATAR_GFX_VSSEEKER,
+};
```

Comment:
These are the FRLG equivalent constants to RSE's `PLAYER_AVATAR_STATE`. The latter should be removed here altogether (their uses in the `PLAYER_AVATAR_FLAG_` defines are also incorrect, they should just be the numbers 0-7 there)

Reply:
Right, I was going to get around to fixing that.

Reply:
These are indeed two completely separate enums. Names of both have been fixed.